istioctl/cmd/authz.go

```diff
@@ -246,32 +236,32 @@ func newConverter(v1PolicyFiles, serviceFiles []string, istioNamespace, istioMes
 	return converter.New(k8sClient, v1PolicyFiles, serviceFiles, meshConfig, istioNamespace, istioMeshConfigMapName)
 }
 
-func newValidator(policyFiles []string) (*auth.Validator, error) {
+func newValidator(policyFiles []string) (*authz.Validator, error) {
 	if len(policyFiles) == 0 {
 		return nil, fmt.Errorf("no input file provided")
 	}
-	validator := &auth.Validator{
+	validator := &authz.Validator{
 		PolicyFiles:          policyFiles,
 		RoleKeyToServiceRole: make(map[string]model.Config),
 	}
 	return validator, nil
 }
 
-// Auth groups commands used for checking the authentication and authorization policy status.
+// AuthZ groups commands used for inspecting and interacting the authorization policy.
 // Note: this is still under active development and is not ready for real use.
-func Auth() *cobra.Command {
+func AuthZ() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "auth",
-		Short: "Inspect and interact with authentication and authorization policies in the mesh",
-		Long: `Commands to inspect and interact with the authentication (TLS, JWT) and authorization (RBAC) policies in the mesh
-  check - check the TLS/JWT/RBAC settings based on the Envoy config
+		Use:   "authz",
+		Short: "Inspect and interact with authorization policies",
+		Long: `Commands to inspect and interact with the authorization policies
+  check - check the authorization configuration in Envoy
   convert - convert v1alpha1 RBAC policies to v1beta1 authorization policies
-	validate - check for potential incorrect usage in authorization policy files.
+  validate - validate v1alpha1 RBAC policies for potential incorrect usage
 `,
-		Example: `  # Check the TLS/JWT/RBAC settings for pod httpbin-88ddbcfdd-nt5jb:
+		Example: `  # Check the authorization configuration for pod httpbin-88ddbcfdd-nt5jb:
   istioctl experimental auth check httpbin-88ddbcfdd-nt5jb
 
-  # Convert the v1alpha1 RBAC policies currently applied in the cluster to v1beta1 authorization policies:
+  # Convert the v1alpha1 RBAC policies in the current cluster to v1beta1 authorization policies:
   istioctl experimental auth convert > v1beta1-authz.yaml
```

Comment:
Could you use `x` instead of `experimental` to be consistent? Same with some other places.